src/gt4sd/algorithms/generation/diffusion/implementation.py

```diff
@@ -45,9 +45,14 @@
     ScoreSdeVeScheduler,
     StableDiffusionPipeline,
 )
+from packaging import version
 
 from ....frameworks.torch import device_claim
 
+OLD_DIFFUSERS = version.parse(importlib_metadata.version("diffusers")) < version.parse(
```

Comment:
May I suggest `DIFFUSERS_VERSION_LT_0_6_0`?
```suggestion
DIFFUSERS_VERSION_LT_0_6_0 = version.parse(importlib_metadata.version("diffusers")) < version.parse(
```